UI/Selenium/Steps/CheckAlertsCommandCenterSteps.cs

```diff
@@ -308,55 +294,45 @@ public void ThenTheTheVideoHearingsOfficerSeeTheAlertFailedSelf_TestNoToMicropho
         [Then(@"the the Video Hearings Officer see the alert Failed self-test \(No to Video\) participant F & L name")]
         public void ThenTheTheVideoHearingsOfficerSeeTheAlertFailedSelf_TestNoToVideoParticipantFLName()
         {
+            Driver = GetDriver(_hearing.Participant.Where(a => a.Role.Name.ToLower() == "vho").FirstOrDefault().Id, _scenarioContext);
             _hearing = (Hearing)_scenarioContext["Hearing"];
             var alerts = Driver.FindElements(SelectYourHearingListPage.FailedAlert);
-            Assert.IsTrue(alerts.Count > 0);
-
+            alerts.Count.Should().BeGreaterThan(0);
```

Comment:
Same again, is it always 1 alert or always 2, or a changing amount?

Reply:
see above